README.md

```diff
@@ -161,6 +161,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Webpage Nepal Pvt. Ltd.](http://webpagenepal.com/) | New Road, Pokhara | Pokhara based web development company
 - [WebSoft Nepal Pvt. Ltd.](http://www.websoftnepal.com) | Indradev Marg, Narayangarh | Provides a complete solution on Web Design and Hostings
 - [Worldlink Communications Pvt. Ltd.](https://worldlink.com.np/) | Jawalakhel, Lalitpur | Complete IT service provider providing services such as data connectivity, network integration and consulting, web hosting, hardware sales and maintenance
+- [WebRobo](https://www.web-robo.com/) | Maitidevi, Setopul, Kathmandu, Nepal | Creative Website development and digital marketing
```

Comment:
@akritibhusal Hey, thank you for the contribution. 

The location needs to be stripped down so that it is consistent. Maybe `Maitidevi, Kathmandu` or `Setopul, Kathmandu`.

Reply:
Sure, let me fix that.